org.xonotic.Xonotic.json

```diff
@@ -14,47 +14,42 @@
         "--persist=.xonotic"
     ],
     "command": "xonotic-sdl",
-    "cleanup": [
-        "/include",
-        "/lib/pkgconfig",
-        "/share/aclocal",
-        "/share/man",
-        "*.la", "*.a"
-    ],
     "modules": [
+        /* See: https://gitlab.com/xonotic/xonotic/-/wikis/Compiling */
         {
             "name": "xonotic",
             "buildsystem": "simple",
             "build-options": {
                 "env": {
-                    "PREFIX": "/app"
+                    "prefix": "/app", /* this and several other GNU make standard directory variables are supported */
+                    "BASEDIR": "/app/share/xonotic", /* eliminates $PATH symlinks by telling engine where data/*.pk3 is */
+                    "FILENAME_DESKTOP_CLIENT": "org.xonotic.Xonotic", /* flathub requires desktop and icon files be renamed */
+                    "FILENAME_ICON_CLIENT": "org.xonotic.Xonotic", /* no file extensions here! */
+                    "CFLAGS": "", /* disables -march=native (all other optimisations enabled by default as in upstream builds) */
+                    "RANLIB": "gcc-ranlib", /* fixes LTO build on Freedesktop SDK 24.08 ?not needed on debian, gentoo, etc? */
+                    /* legacy engine requires following kludges to build with current Makefile, RELEASE TODO: remove for 0.9 */
+                    "DP_LINK_CRYPTO": "foo",
```

Comment:
What's this used for?

Reply:
It makes player IDs and xonstat work (the legacy engine in 0.8.6 has bugs in its Makefile requiring this, without it the library doesn't get linked).